tests/unit/ValidationTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Facades\Validator;
+
+class ValidationTest extends TestCase
+{
+    /**
+     * A greater than validation rule test.
+     *
+     * @return void
+     */
+    public function testGreaterThanRule()
+    {
+        $validator = Validator::make([
```

Comment:
Don't they all return void? If so many all the comments should reflect it. Not just this?